assets/src/styles/pages/_monitoring-location.scss

```diff
@@ -18,6 +18,10 @@
   @include u-padding-bottom(1);
 }
 
+.iv-graph-data-download-container {
```

Comment:
This style is for a class, but line 18 in the macro shows an id. Either this is wrong or you have a class with the same name as an id. I'm not crazy about the latter.

Reply:
Wow. Good catch. I think this was a leftover from some changes that were later changed (again). I removed the lines, since they are no longer needed.